client/pages/homepage/carousel-component.js

```diff
@@ -0,0 +1,50 @@
+const CarouselComponent = {
+  template: `
+  <div class="homepage__carousel">
+      <h2 class="carousel__title">see what we have achieved as HYF community!</h2>
+      <h3 class="carousel__subtittle">and many more projects to build...</h3>
+      <div class="container">
```

Comment:
Change name of this class to: homepage-carousel-container to avoid confusion

Reply:
Ok

Reply:
Done! I changed it to `homepage__carousel--container` to keep the same style of the classes in homepage, except for bootstrap classes, of course.